core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1503,10 +1535,9 @@ public void reportLostFile(long fileId) throws FileDoesNotExistException {
    * @throws AccessControlException if permission checking fails
    */
   // TODO(jiri): Make it possible to load UFS objects recursively.
```

Comment:
I didn't address your TODO completely since you wanted to load UFS objects recursively. We are only loading the direct children in this PR. 

 I have removed the TODO for now.